tests/system/action/user/test_create.py

```diff
@@ -377,6 +377,21 @@ def test_create_strip_spaces(self) -> None:
             },
         )
 
+    def test_create_smal_id_and_default_pasword(self) -> None:
```

Comment:
write a test with saml_id set and empty default_password. Assert an empty_passwort, can_change_own_password False and no password set.